### PR TITLE
Enable ETW logger when using AppServices integration

### PR DESCRIPTION
**Pull request by pakrym**
https://github.com/aspnet/AzureIntegration/issues/134

### CODE REVIEW

**Review by muratg — APPROVED** [review]

**Review by shirhatti — APPROVED** [review]

**Comment by davidfowl** [comment]
We need to figure out what happens if we ever decide to turn this on by default.

**Comment by davidfowl** [comment]
If we call `AddEventSourceLogger` by default in CreateDefaultBuilder in 3.0 say. This will be fine right? It'll just noop.

**Comment by anurse** [comment]
> If we call `AddEventSourceLogger` by default in CreateDefaultBuilder in 3.0

What if we just do that in 2.2? 😉

File: src/Microsoft.AspNetCore.AzureAppServicesIntegration/AppServicesWebHostBuilderExtensions.cs [hunk 1]
```diff
@@ -19,9 +20,20 @@ public static IWebHostBuilder UseAzureAppServices(this IWebHostBuilder hostBuild
...
-            hostBuilder.ConfigureLogging(builder => builder.AddAzureWebAppDiagnostics());
-#pragma warning restore 618
+            hostBuilder.ConfigureLogging(builder => builder
+                .AddAzureWebAppDiagnostics()
+                .AddEventSourceLogger()
+            );
+            hostBuilder.ConfigureServices(collection => collection.Configure<LoggerFilterOptions>(
+                options =>
+                {
+                    // Enable event source logger provide for all categories at Information level
```
Comment (anurse) [inline src/Microsoft.AspNetCore.AzureAppServicesIntegration/AppServicesWebHostBuilderExtensions.cs]: Only Information? In theory, the EventSource logger has it's own filtering mechanic so maybe we should just pump all messages through to it.

Comment (davidfowl) [reply src/Microsoft.AspNetCore.AzureAppServicesIntegration/AppServicesWebHostBuilderExtensions.cs]: Yes, I was playing with this last night. I don't know why we need this. Can't this be controlled from the outside? How does the end to end work here?

Comment (pakrym) [reply src/Microsoft.AspNetCore.AzureAppServicesIntegration/AppServicesWebHostBuilderExtensions.cs]: https://github.com/aspnet/Logging/issues/877

Filters are applied before messages get to the event logger. We decided to let only info through for 2.2 and fix end to end for 3.0

Comment (anurse) [reply src/Microsoft.AspNetCore.AzureAppServicesIntegration/AppServicesWebHostBuilderExtensions.cs]: Yeah, we need to revisit the EventSourceLogger in 3.0 for a few reasons if we're going to make it part of an end-to-end diagnostics egress story.

**Review by davidfowl — CHANGES_REQUESTED** [review]
OK After talking to @pakrym I think we shouldn't do this and instead, we should change the meta package to turn this on by default *and* we should fix the EventSourceLoggerProvider itself so that is isn't disabled by default.

**Comment by anurse** [comment]
> we should change the meta package to turn this on by default _and_ we should fix the EventSourceLoggerProvider itself so that is isn't disabled by default.

🎉

**Pull request closed without merging**